tools/cloneall

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+#
+# Run this script from your $HOME/zot/dev directory and ensure you already have utils cloned
```

Comment:
Have you considered adding this script to utils instead?  
Something like:

Clone all:
`zopen clone all`

Clone a specific project
`zopen clone make`